src/components/deviceMap/clusterIcons.js

```diff
@@ -8,10 +8,11 @@ function onlineWarn(radius, clusterSize) {
     radius * 2,
     '" height="',
     radius * 2,
+
     '">',
     '<rect width="24" x="0" y="5" height="24" style="fill:rgb(255,255,255);stroke-width:2;stroke:rgb(175,185,195)" />',
     '<rect width="8" x="18" y="0" height="8" fill="#FDE870" />',
-    '<text x="10" y="18" text-anchor="middle" alignment-baseline="central" font-size="12" style="fill:black">' +
+    '<text x="10" y="18" text-anchor="middle" alignment-baseline="central" font-size="12" font-family="Helvetica Neue,Helvetica,Arial,sans-serif" style="fill:black">' +
```

Comment:
Is defining the font family explicitly really the only option? `inherit` doesn't work? This doesn't seem to be good for future updates.

Reply:
Inherit is not working that's i have to add it in all svg's.